heliopy/data/util.py

```diff
@@ -176,6 +177,41 @@ def units_attach(data, units):
     return timeseries_data
 
 
+def cdf_units(cdf_, keys=None):
+    """
+    Takes the CDF File and the required keys, and finds the units of the
+    selected keys.
+
+    Parameters
+    ----------
+    cdf_ : cdf
+        Opened cdf file
+    keys : :class 'dict'
+        The keys required by user.
+
+    Returns
+    -------
+    out : :class:`collections.OrderedDict`
```

Comment:
I wonder why is this failing CircleCI. The units_attach is working perfectly well.